src/herder/Upgrades.cpp

```diff
@@ -393,7 +393,7 @@ Upgrades::toString(LedgerUpgrade const& upgrade)
     case LEDGER_UPGRADE_CONFIG:
         return fmt::format(
             FMT_STRING("{}"),
-            xdr::xdr_to_string(upgrade.newConfig(), "configupgradesetkey"));
```

Comment:
This was confusing to figure out. We have two different `xdr_to_string` methods. One in [xdrpp](https://github.com/xdrpp/xdrpp/blob/8756126cc734c11b75e05f174ca35a192860d0a6/xdrpp/printer.h#L162), and one in [XDRCereal.h](https://github.com/stellar/stellar-core/blob/master/src/util/XDRCereal.h#L186). I added a `cereal_override` but it did nothing because we were using the xdrpp methods. Maybe we should rename, add a namespace, or stop using the xdrpp method altogether.